README.md

```diff
@@ -7,14 +7,46 @@ P. Ramadoss, L.Rapetti, Y.Tirupachuri, R.Grieco, G. Milani, E. Valli, S. Dafarra
 </i>
 </div>
 
+<p align="center">
+
+
+[ICHR 2022 - Estimation of Human Base Kinematics](https://user-images.githubusercontent.com/6506093/204252819-0eb9e7fe-abf5-46db-9fb9-9df48aff5a20.mp4)
+
+
+<div align="center">
+2022 International Conference on Humanoid Robotics (Humanoids 2022)
+</div>
+
+<div align="center">
+  <a href="#reproducing-the-experiments"><b>Installation</b></a> |
+  <a href="https://arxiv.org/abs/2205.07835"><b>arXiv</b></a> | 
+  <a href="https://youtu.be/svX_QNSaElw"><b>YouTube</b></a>
+</div>
 
 
 ## Reproducing the experiments
 
-To be updated soon.
+We provide a containerised virtual environment using Docker and Conda in order to launch the software in an isolated, reproducible manner. The dependencies and related versions used to build the environment can be checked in the `deps` folder. However, one can simply pull the pre-built docker image from the GitHub registry, since a `docker.yaml` in the `.github/workflows` folder dispatches a build workflow for the docker image.
+
+1. Pull the docker image:
+    ```bash
+    docker pull ghcr.io/ami-iit/human-base-estimation-docker:latest
+    ```
+
+2. Launch the container:
+   ```bash
+   xhost +
+   docker run -it --net=host --env="DISPLAY=$DISPLAY"  --volume="/tmp/.X11-unix:/tmp/.X11-unix" ghcr.io/ami-iit/human-base-estimation-docker:latest
+   ```
+
+   Please be aware that `xhost +` is not a safe way to expose the X Server running on the host machine to the docker container. But in this scenario,  we are not doing anything undesirable, so it is acceptable.
+
+3. The experiment will start by automatically launching a `yarp server`, loading the dataset  and launching the necessary applications along with the visualizer. The experiment will end and everything will close automatically.
 
```

Comment:
@prashanthr05 Do you think adding a codespace to the repo to run things there can be useful ?

Reply:
Would you have some tutorials or documentation references on how to do that?

Reply:
https://docs.github.com/en/codespaces/developing-in-codespaces/creating-a-codespace-for-a-repository

Reply:
Please double check that you are not using `ami-iit`'s organization codespace credits (or at least, not using them too much). If I remember correctly Codespaces credit are quite limited, even when used with free repos.

Reply:
Then, I think we may avoid using them. If one has docker, it must be straight-forward to reproduce the expected results of this repository.